assets/config.php

```diff
@@ -98,10 +98,10 @@
      */
     'bootstrappers' => [
         Stancl\Tenancy\Bootstrappers\DatabaseTenancyBootstrapper::class,
-        Stancl\Tenancy\Bootstrappers\CacheTenancyBootstrapper::class,
```

Comment:
Did we get rid of the tags-based bootstrapper completely?

Reply:
There isn't any use for it anymore. It just swapped the CacheManager instance, but since we're doing that in TSP now, the bootstrapper doesn't really do anything (explained here: https://github.com/archtechx/tenancy/pull/1014/files#r1154378154)

Reply:
Marking as resolved since we continued the convo in the other thread & Discord.